docs/docs/api-reference/starlark-reference/plan.md

```diff
@@ -570,6 +570,15 @@ The `run_python` instruction executes a one-time execution task. It runs the Pyt
             "/coinbase/address.json"
         ],
 
+        # If the recipe returns a code that does not belong on this list, this instruction will fail.
```

Comment:
maybe "that does not belong on" -> "that isn't a part of", and "this instruction will fail" -> "this instruction will fail at execution time"?